.editorconfig

```diff
@@ -0,0 +1,3 @@
+# Tab indentation
+[*]
+indent_style = tab
```

Comment:
This addition will come in a separate PR. This slipped in here as I wanted to set this as I was working through this :)

https://github.com/hashicorp/dev-portal/pull/756